traffic_monitor_golang/traffic_monitor/datareq/datareq.go

```diff
@@ -157,16 +159,34 @@ func WrapErrCode(errorCount threadsafe.Uint, reqPath string, body []byte, err er
 // WrapBytes takes a function which cannot error and returns only bytes, and wraps it as a http.HandlerFunc. The errContext is logged if the write fails, and should be enough information to trace the problem (function name, endpoint, request parameters, etc).
 func WrapBytes(f func() []byte, contentType string) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		bytes := f()
+		bytes, err := gzipIfAccepts(r, w, bytes)
+		if err != nil {
```

Comment:
Shouldn't this error be handled the same as others? I think it should increment errorCount, like all the other routines, right?

And if it's doing that, why wouldn't it just be a wrapper around WrapErr, which contrary to the indication one might glean from it's name is just the same as this, but with errors as well. In fact, this function might could be written:

    return WrapErr( errorCode /* that probably needs to be passed in */,
        func() ([]byte, err) { return f(), nil },
        contentType)